packages/cbioportal-utils/src/api/apiUtils.ts

```diff
@@ -8,3 +10,69 @@ export async function chunkCalls<ParamData, ResponseData>(
     const proms = chunks.map(callback);
     return _.flatten(await Promise.all(proms));
 }
+
+function monkify(obj: any) {
+    return Buffer.from(_.isString(obj) ? obj : JSON.stringify(obj)).toString(
+        'base64'
+    );
+}
+
+function unmonkify(base64String: string) {
+    return JSON.parse(Buffer.from(base64String, 'base64').toString('utf8'));
+}
+
+export function maskApiRequests(apiClient: any, domain: string) {
+    const oldRequestFunc = apiClient.prototype.request;
+
+    apiClient.prototype.request = (
+        method: string,
+        url: string,
+        body: any,
+        headers: any,
+        queryParameters: any,
```

Comment:
Should we enforce this at least to be array|object?

Reply:
Ideally yes. But this is inline with the function that we override:

https://github.com/cBioPortal/cbioportal-frontend/blob/fa86ca51b2a535aad89c2428270e35f1068e75f6/packages/oncokb-ts-api-client/src/generated/OncoKbAPI.ts#L375